README.md

```diff
@@ -18,6 +18,15 @@ PRISM Break follows [the GNU/FSF definition of Free Software](https://www.gnu.or
 
 PRISM Break strives to promote the best open source applications. Ease of use, stability, and performance matter. This is the first time many people are looking to leave their proprietary walled gardens. Let's make it a good experience for them. If you're writing a privacy-minded FOSS app, please finish it before asking PRISM Break to promote it.
 
+ - **The software must actually be secure.**
+
+This can vary case-to-case, but generally here are the attributes we look for:
+
+ ** Understand security and take it seriously (this is very broad, but can range from "don't invent your own crypto" to "think about and design against metadata leaks")
+ ** Offers security advantages that other projects don't
+ ** Can keep up with security events
+ ** Doesn't depend on, imply or promote insecure things including and especially things in the "proprietary" column
```

Comment:
Don't confuse insecure with proprietary. I'd guess a lot of proprietary software is actually more secure than some opensource counterparts (if there are any, often there is not for industry grade tools). We just don't want it because of **trust**. I'm not sure we really need to say again that we don't like proprietary software. That's already established.

Reply:
Right, that's what I meant but I was conflating the language. In any case I think you're right that this is just cluttering up the list; will remove